docs/modding/squirrel/gamemode-mods/the-mod-init.md

```diff
@@ -0,0 +1,136 @@
+Creating Gamemodes
+==================
```

Comment:
AFAIK, gitbook solely uses `#` for headings, so probably best to change from

```
Heading 1
========
```

to

```
# Heading 1
```

and

```
Heading 2
-------------
```
to
```
## Heading 2
```


Additionally, if H1 heading and filename do not match up, gitbook will add filename as H1 heading (so in this case `# The Mod Init`) once the file is modified on there. Not something you need to necessarily change right now but something you might want to be aware of.